src/tests/Microsoft.PowerFx.Interpreter.Tests.Shared/RecalcEngineTests.cs

```diff
@@ -798,6 +798,43 @@ public void DelegableUDFTest()
 
             // Binding fails for recursive definitions and hence function is not added.
             Assert.False(recalcEngine.AddUserDefinedFunction("E():Void = { E(); };", CultureInfo.InvariantCulture, symbolTable: recalcEngine.EngineSymbols, allowSideEffects: true).IsSuccess);
+        }
+
+        [Fact]
+        public void TestInheritanceOfDelegationWarningsInUDFs()
```

Comment:
>TestInheritanceOfDelegationWarningsInUDFs [](http://example.com/codeflow?start=20&length=41)

Can we have a test for the logic on Named Formulas?

Reply:
Will add integrated test in PA client for NF scenario